pkg/dao/schema/workflow.go

```diff
@@ -58,6 +59,9 @@ func (Workflow) Fields() []ent.Field {
 			Default(0).
 			Annotations(
 				entx.SkipInput()),
+		field.JSON("variable_configs", types.VariableConfigs{}).
```

Comment:
What about name this field "variables" and the type "WorkflowVariable"? I wonder if we need another concept since we already used this term in other places like template schema.

Reply:
updated to 
```
field.JSON("variables", types.WorkflowVariables{}).
			Comment("Configs of workflow variables.").
			Optional(),

```